test/UnitConversions/Tests.hs

```diff
@@ -17,11 +19,27 @@ isMetric unit =
     _  -> False
 
 tests :: TestTree
-tests = testGroup "UnitConversions"
+tests = testGroup "IngredientConversions"
     [ testProperty "testToMetric" $
         \x -> isMetric (unit x) ==>
                 convertIngredientToMetric x === x
     , testProperty "testToImperial" $
         \x -> not (isMetric (unit x)) ==>
                 convertIngredientToImperial x === x
     ]
+
```

Comment:
I'm happy to see you've added tests! A good QuickCheck property for these functions might be the following: Converting to F (resp. C) and back changes the original magnitude by no more than 2%.

You can add this if you feel like it, but it won't hold up merging :smile:

Reply:
I have to pass on this. I tried for a bit but noticed that I need to get deeper into QuickCheck to write a useful test.

Reply:
No worries! Perhaps I'll find the time to do it later.